OneBlink.SDK/JobsClient.cs

```diff
@@ -0,0 +1,107 @@
+using System.Threading.Tasks;
+using OneBlink.SDK.Model;
+using Amazon;
+using Amazon.Runtime;
+using Amazon.S3;
+using Amazon.S3.Model;
+using Newtonsoft.Json;
+using System;
+
+namespace OneBlink.SDK
+{
+    public class JobsClient
+    {
+        OneBlinkHttpClient oneBlinkHttpClient;
+
+        public JobsClient(string accessKey, string secretKey)
+        {
+            this.oneBlinkHttpClient = new OneBlinkHttpClient(accessKey, secretKey);
+        }
+
+        public async Task DeleteJob(string jobId)
+        {
+            if (string.IsNullOrWhiteSpace(jobId))
+            {
+                throw new ArgumentException("'jobId' must be provided with a valid value");
+            }
+
+            string url = $"/jobs/{jobId}";
+
+            await this.oneBlinkHttpClient.DeleteRequest(url);
+        }
+
+        public async Task<Job> CreateJob(JobDetail details, string externalId, int formId, string username)
```

Comment:
```suggestion
        public async Task<Job> CreateJob(JobDetail details, int formId, string username, string externalId)
```

Reply:
Can we add another method the same as this one but without the `externalId` parameter for convenience